### ISSUE TITLE
1. User: Local User

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#966]
- [ ] Workflow
  - [ ] [click "Continue Offline" in the User Screen] > Terms and Conditions > Create Username > Warning modal > Project
- [ ] Terms and Conditions
  - [ ] Same as the Terms and Conditions for creating a Door43 account, except the wording of the first phrase is changed
  - [ ] All links should be available offline in modal pop-ups (see #965 )
- [ ] Create User Profile
  - [ ] "Continue" button is gray and inactive until something is entered into the "username" field, then the button turns blue and is active
  - [ ] When "Continue" is clicked, go to the "Attention" pop-up
- [ ] "Attention" Pop-up
  - [ ] If "Go back" is selected, it returns the user to the "Create User Profile" screen
  - [ ] If "Confirm" is selected, take the user to "Project" screen

**Comment by Thajny** [comment]
![welcome 8](https://cloud.githubusercontent.com/assets/21183377/24414370/e03d0074-13a3-11e7-8292-bf84590d323d.png)
![welcome 9](https://cloud.githubusercontent.com/assets/21183377/24414371/e03f1fb2-13a3-11e7-9e5c-8436628b5099.png)
![welcome 15](https://cloud.githubusercontent.com/assets/21183377/24414372/e043e7a4-13a3-11e7-9384-9fe095db6147.png)

**Comment by Thajny** [comment]
![offline user-1](https://cloud.githubusercontent.com/assets/21183377/24562621/f61c5c04-1610-11e7-9854-983492e1053c.png)
![offline user-2](https://cloud.githubusercontent.com/assets/21183377/24562623/f61fc448-1610-11e7-9cf9-155ce05dc558.png)
![offline user](https://cloud.githubusercontent.com/assets/21183377/24562622/f61d420e-1610-11e7-9228-bbcd4c8f9923.png)

**Comment by Thajny** [comment]
![offline user-1](https://cloud.githubusercontent.com/assets/21183377/24562878/feabfa36-1611-11e7-9e3e-f00acb8f92aa.png)
![offline user-2](https://cloud.githubusercontent.com/assets/21183377/24562880/fecbf64c-1611-11e7-9c6c-60dce807f72c.png)
![offline user](https://cloud.githubusercontent.com/assets/21183377/24562877/feab9e1a-1611-11e7-856e-c657abcf9379.png)

**Comment by Thajny** [comment]
![new-terms -1](https://cloud.githubusercontent.com/assets/21183377/24664368/1b4ab324-1920-11e7-85f7-dcdd6aa7b424.png)
![new-terms 1](https://cloud.githubusercontent.com/assets/21183377/24664369/1b4f440c-1920-11e7-8561-eb7023ba0041.png)

**Comment by Thajny** [comment]
![offline user_modal](https://cloud.githubusercontent.com/assets/21183377/24667813/4b203e48-192a-11e7-88ea-d74aa02d892d.png)

**Comment by Thajny** [comment]
![local account](https://user-images.githubusercontent.com/21183377/27202312-aca3bd80-51e6-11e7-8e34-f52720145c35.png)
![terms modal 1](https://user-images.githubusercontent.com/21183377/27202336-c98a9f22-51e6-11e7-9b0e-9274fa670b1e.png)
![terms modal](https://user-images.githubusercontent.com/21183377/27202337-c98ccbc6-51e6-11e7-904e-9ab1254ec09b.png)

**Comment by Thajny** [comment]
User is creating a local account, but terms and conditions refer to creating a Door43 account.

![local account 2](https://user-images.githubusercontent.com/21183377/27207618-ca8d3f50-5205-11e7-9ab5-bfe2ea05a6f9.png)
![local account](https://user-images.githubusercontent.com/21183377/27207620-ca8fa9b6-5205-11e7-8454-acce8fd2d6ce.png)
![terms modal 1](https://user-images.githubusercontent.com/21183377/27207621-ca91d36c-5205-11e7-923b-69f39b1b1f92.png)
![terms modal](https://user-images.githubusercontent.com/21183377/27207619-ca8f5358-5205-11e7-8c96-d22ea2651da9.png)

**Comment by mannycolon** [comment]
## Task
- [ ] [2] Create `reusable local user modal` for local user contents.
- [ ] [2] Move local user page from current implementation to Card implementation. 
- [ ] [1] Move content to `reusable local user modal`

**Comment by cckozie** [comment]
1. The Create button goes active after checking the checkbox even if no username has been entered.
2. The text on the terms and conditions dialog is for a door43 account, not a local user account.
3. The text in the title bar for Terms and Conditions should be white
4. The Next button should be disabled until the user logs in
5. The gammer on the interim dialog is bad. Should be "You have chosen..." and "This username will be publicly viewable."
![image](https://user-images.githubusercontent.com/8171759/27740109-6b03b416-5d76-11e7-9488-91c70c9347ab.png)
6. There are two Go Back buttons on this screen, and they take the user to different places. Is that the desired behavior?
![image](https://user-images.githubusercontent.com/8171759/27740383-763eec5a-5d77-11e7-8e86-3eac161e9c8f.png)

**Comment by benjore** [comment]
Verbiage for the "Terms and Conditions"

By continuing, you affirm that you...
Only add content that does not infringe on someone else's copyright.
Only add content that does not conflict with the Statement of Faith. 
Release your contributions under a ...

**Comment by benjore** [comment]
As for 6. , The main navigation buttons on the bottom should say where the button will take the user. So in the screen shot above it should be "Go Home" and "Go to Projects" (when in Projects is should say "Go to User" and "Go to Tools"). The forward action should be disabled until all prerequisite actions have been completed (e.g. login complete).

**Comment by mannycolon** [comment]
working on `QA fail`

**Comment by mannycolon** [comment]
@benjore @cckozie Number 6 should be its own story. I will go ahead and creat it.

**Comment by mannycolon** [comment]
1. Done
2. Done
3. Done
4. Done
5. Done
6. Will be addressed by another story.

**Comment by cckozie** [comment]
1. OK
2. OK
3. OK
4. The next button is not disabled. I am allow to proceed to selecting a project and tool without first logging in. It is not until I have selected the tool that I am stopped.
5. OK
6. OK

**Comment by cckozie** [comment]
I see that #2027 addresses issue 4 so I will pass this one.
